packages/hooks/src/useLocalStorageState/index.en-US.md

```diff
@@ -40,11 +40,12 @@ const [state, setState] = useLocalStorageState<T>(
 
 ### Options
 
-| Property     | Description                   | Type                     | Default          |
-| ------------ | ----------------------------- | ------------------------ | ---------------- |
-| defaultValue | Default value                 | `any \| (() => any)`     | -                |
-| serializer   | Custom serialization method   | `(value: any) => string` | `JSON.stringify` |
-| deserializer | Custom deserialization method | `(value: string) => any` | `JSON.parse`     |
+| Property     | Description                   | Type                       | Default                    |
+| ------------ | ----------------------------- | -------------------------- | -------------------------- |
+| defaultValue | Default value                 | `any \| (() => any)`       | -                          |
+| serializer   | Custom serialization method   | `(value: any) => string`   | `JSON.stringify`           |
+| deserializer | Custom deserialization method | `(value: string) => any`   | `JSON.parse`               |
+| onError      | On error callback             | `(error: unknown) => void` | `(e) => {console.error(e)` |
 
```

Comment:
the default value `(e) => {console.error(e)` has a syntax error, should be: `(e) => { console.error(e) }`, right?

Reply:
I'm sorry

Reply:
Modified